hw1/02-palindrome.js

```diff
@@ -1,3 +1,19 @@
 const elem = document.querySelector('input');
+const displayOutput = document.getElementById('outputMessage');
+
+const handleInput = () => {
+	displayOutput.style.color = 'red';
+	let outputMessage = '';
+	let { value } = elem;
```

Comment:
_Style_ - Airbnb's style guide prefers `const` when variables aren't reassigned, which applies to `value` (reference: [rule 2.1](https://github.com/airbnb/javascript#variables)).